scripts/zkai_d128_native_block_gap_accounting_gate.py

```
@@ -0,0 +1,852 @@
+        for index, (path, contents) in enumerate(outputs, start=1):
+            label = f"output path {index}"
+            temp = write_temp(path, contents, label)
+            temps.append(temp)
+            replace_temp(temp, path, label)
+            replaced.append(path)
+        for _temp_path, temp_name, parent_fd, _identity in temps:
+            try:
+                os.unlink(temp_name, dir_fd=parent_fd)
+                os.fsync(parent_fd)
+            except FileNotFoundError:
+                pass
```

Ruff (0.15.12)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)